e2e_tests/tests/cluster/test_users.py

```diff
@@ -584,7 +570,11 @@ def test_non_admin_commands(clean_auth: None, login_admin: None) -> None:
         assert child.exitstatus != 0
 
 
-def run_command() -> str:
+def run_command(session: api.Session) -> str:
+    if session:
```

Comment:
we could probably have a thing that reads from token store into a session to make these sort of transitions easier if we wanna go this way

Reply:
When would `session` be `None`?  The typing suggests it can't be.

Reply:
Looks like this function is only used in this file, so it should be safe to switch over entirely.

Reply:
yeah, I'm not sure what lead to this. I simplified it.